### PR TITLE
removed SQL query from code into ddl folder cmslitemetadata_to_redshift

### PR DESCRIPTION
**Pull request by TaylorFries**

### CODE REVIEW

File: cmslitemetadata_to_redshift/cmslite_gdx.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "doc": "cmslite.csv.*",
   "dbschema": "cmslite",
   "dbtable": "metadata",
+  "ddl": "cmslite.to_redshift.sql",
```
Comment (veenupunyani) [inline cmslitemetadata_to_redshift/cmslite_gdx.json:9]: To keep consistency for variable names in all microservices, can we use same name"sql_query" here and probably use it in a same way. Open to discussion on this

URL - https://github.com/bcgov/GDX-Analytics-microservice/blob/main/s3_to_redshift/README.md

File: cmslitemetadata_to_redshift/cmslite_groups_gdx.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "doc": "cmslite.csv.*",
   "dbschema": "cmslite",
   "dbtable": "metadata",
+  "ddl": "cmslite.to_redshift.sql",
```
Comment (veenupunyani) [inline cmslitemetadata_to_redshift/cmslite_groups_gdx.json:9]: To keep consistency for variable names in all microservices, can we use same name"sql_query" here and probably use it in a same way. Open to discussion on this

URL - https://github.com/bcgov/GDX-Analytics-microservice/blob/main/s3_to_redshift/README.md

File: cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py [hunk 3]
```diff
@@ -92,6 +92,7 @@ def clean_exit(code, message):
     dbtables_metadata = data['dbtables_metadata']
     nested_delim = data['nested_delim']
     columns = data['columns']
+    ddl_file = data['ddl']
```
Comment (veenupunyani) [inline cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py:95]: https://github.com/bcgov/GDX-Analytics-microservice/blob/1f96d2e846a3e6651f8092712f910aa4d842c6c2/s3_to_redshift/s3_to_redshift.py#L111

File: cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py [hunk 4]
```diff
@@ -575,434 +576,8 @@ def report(data):
...
-       asset_subtopic,
-       asset_subsubtopic,
-       sitekey
-FROM biglist
-WHERE index = 1;
-
-ANALYZE {dbschema}.asset_themes;
-
-    """.format(dbschema=dbschema)
+    query = open('ddl/{}'.format(ddl_file), 'r').read()
```
Comment (veenupunyani) [inline cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py:579]: https://github.com/bcgov/GDX-Analytics-microservice/blob/1f96d2e846a3e6651f8092712f910aa4d842c6c2/s3_to_redshift/s3_to_redshift.py#L569-L571

**Review by veenupunyani — CHANGES_REQUESTED** [review]
open to discussion on these let me know

File: cmslitemetadata_to_redshift/cmslite_gdx.json (hunk 1 above)
Comment (TaylorFries) [reply cmslitemetadata_to_redshift/cmslite_gdx.json]: added commit to match these standards!

File: cmslitemetadata_to_redshift/cmslite_groups_gdx.json (hunk 2 above)
Comment (TaylorFries) [reply cmslitemetadata_to_redshift/cmslite_groups_gdx.json]: added commit to match these standards

File: cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py (hunk 3 above)
Comment (TaylorFries) [reply cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py]: changed this to something that will work better for the code as we dont want the script to run if this is not set:
`if 'sql_query' not in data:
        clean_exit(1, "bad configuration")
    ddl_file = data['sql_query']`
Let me know if this doesnt work!

File: cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py (hunk 4 above)
Comment (TaylorFries) [reply cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py]: added a portion of this, but there is no way (in this script) to get to this point if the outfile is not goodfile 
Let me know if this doesnt make sense

**Review by veenupunyani — APPROVED** [review]

**Pull request merged**